### DISCUSSION TITLE
speedbrake ring too faint

### DISCUSSION THREAD

**Discussion by crazyrolf** [flybywiresim/aircraft#3632] [General]
The white color from the speedbrake ring in the new cockpit is too faint, very hard to see.

**Comment by MapleNAV-TCASFAULT** [comment]
It looks right to me. Feel free to move this as an issue/request, especially if you have any photo reference to how it should actually look like.

**Comment by hotshotp** [comment]
exactly, please make an issue and put a reference

**Comment by crazyrolf** [comment]
![speedbrake](https://user-images.githubusercontent.com/8255389/108802112-d2ad0d00-7597-11eb-88e3-d35a6c505844.jpg)
please

**Reply by MapleNAV-TCASFAULT** [reply]
Hmm, I’m sorry but that doesn’t really check out to be a proper reference as it’s from an a330. But I guess it is the same.

**Reply by crazyrolf** [reply]
its the same. but if you think it's good the way it is, then leave it like it is. no problem. I like to help, but OK thanks.

**Reply by MapleNAV-TCASFAULT** [reply]
Just open a GitHub request showing this photo and the FBW team will see if it’s plausible to change ;D
